packages/api-server/api_server/models/rmf_api/task_state.py

```diff
@@ -181,6 +181,7 @@ class TaskState(BaseModel):
         None, description="Which agent (robot) is the task assigned to"
     )
     status: Optional[Status] = None
+    unix_millis_request_time: Optional[int] = None
```

Comment:
I think this should be in `Booking`, as it is part of the task information that does not change throughout the entire task and is related to the task request itself

Reply:
https://github.com/open-rmf/rmf_api_msgs/pull/34 would probably need to be updated as well, https://github.com/open-rmf/rmf_api_msgs/blob/main/rmf_api_msgs/schemas/task_state.json#L110

Reply:
Field added to booking according to suggestion!